www/getting-started.rst

```diff
@@ -2,151 +2,83 @@
 
-SciPy and friends can be used for a variety of tasks:
+Just remember to have fun, make mistakes, and persevere. 
 
-* *NumPy*'s array type augments the Python language with an efficient data
-  structure useful for numerical work, e.g., manipulating matrices.  *NumPy*
-  also provides basic numerical routines, such as tools for finding
-  eigenvectors.
+Where to Write
+==============
 
-* *SciPy* contains additional routines needed in scientific work: for example,
-  routines for computing integrals numerically, solving differential equations,
-  optimization, and sparse matrices.
+`Jupyter <https://jupyter.org/>`_ notebooks combine code, markdown, and more in an interactive setting. They are an excellent tool for learning, collaborating, experimenting, or documenting. Notebooks can run on your local machine, and `MyBinder <https://mybinder.org/>`_ also serves Jupyter notebooks to the browser without the need for anything on the local computer. For example, `MyBinder Elegant Scipy <https://mybinder.org/v2/gh/elegant-scipy/notebooks/master?filepath=index.ipynb>`_ provides an interactive tutorial.
 
-* The matplotlib_ module produces high quality plots. With it you can turn your
-  data or your models into figures for presentations or articles. No need to do
-  the numerical work in one program, save the data, and plot it with another
-  program.
+Jupyter runs by calling to `IPython <https://ipython.org/>`_ behind the scenes, but IPython itself also acts as a standalone tool. A *command-line* of individual statements and returned values, IPython is useful for debugging and experimenting.
 
-* Using IPython_ makes interactive work easy. Data processing, exploration of
-  numerical models, trying out operations on-the-fly allows to go quickly from
-  an idea to a result. See the `IPython site <http://ipython.org/>`__ for many
-  examples.
+Code Editors and IDE's (Integrated Development Environments) facilitate the writing of scripts, packages, and libraries.  These tools handle projects, like SciPy itself, that start to grow larger and more complicated. Separate files can hold frequently used functions, types, variables, and analysis scripts for simpler, more maintainable, and more reusable code.  
 
-* There is a sizeable collection of both generic and
-  application-specific numerical and scientific code, written using
-  Python, NumPy and SciPy. Don't reinvent the wheel, there may already
-  be a pre-made solution for your problem. See
-  :doc:`topical-software` for a partial list.
+Code editors run from minimal, like Window's Notepad or `Emacs <https://www.gnu.org/software/emacs/>`_ , to the fully-featured and customizable, like `Atom <https://atom.io/>`_ , `Visual Studio Code <https://code.visualstudio.com/>`_ , or `PyCharm <https://www.jetbrains.com/pycharm/>`_ . Features include syntax highlighting, the ability to execute code, debugging tools, autocompletion, and project management.
```

Comment:
Silly nitpick: calling Emacs minimal instead of "fully-featured and customizable" is likely to ruffle a few feathers.

Reply:
Maybe then I just remove it from the list?  
I haven't used it since I first started writing code, so I didn't know that much about its features.  
Also, reviewing it now, it doesn't look like a good "Getting Started" editor.

Reply:
Getting started with SciPy: one problem

Using Emacs to help you get started with SciPy: two problems :-)